styles/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+    box-sizing: border-box;
+    max-width: 1440px;
+    margin-left: auto;
+    margin-right: auto;
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+}
+
+body {
+    background: #ffffff;
+}
+.advantages {
+    height: 450px;
+}
+.highlight {
+    width: 100px;
+    height: 4px;
+    margin-bottom: 87px;
+    background: #f5bd41;
+    border: 0;
+}
+.heading {
+    font-weight: 500;
+
+    margin: 120px 567px 20px;
+    text-align: center;
+
+    color: #383838;
+    font-size: 40px;
+}
+/*Var 2*/
+/*.vectors{*/
+/*    display: flex;*/
+/*    justify-content: space-around;*/
+/*}*/
+/*.vectors-img{*/
+/*    margin-bottom: 57px;*/
+/*}*/
+
+/*.img-text{*/
+/*    display: flex;*/
+/*    justify-content: space-around;*/
+/*}*/
+/*.img-text-w{*/
+/*    margin: 0;*/
+/*    font-size: 24px;*/
+/*    line-height: 28px;*/
+/*    text-align: center;*/
+/*    color: #383838;*/
+/*}*/
+.down .img-text-v {
+    display: inline;
+}
+.down {
+    margin-bottom: 20px;
+}
+.img-text {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 28px;
```

Comment:
**Just a recommendation**
it's better to use `line-height` with values **without px**
examples
```
line-height: 2.5;
line-height: 3em;
line-height: 150%;
```